zproject/settings.py

```diff
@@ -1041,6 +1044,12 @@ def get_secret(key):
             'propagate': False,
             'filters': ['skip_200_and_304'],
         },
+        'django.template': {
+            'handlers': ['console'],
+            'filters': ['require_debug_true'],
+            'level': 'DEBUG',
+            'propagate': False,
```

Comment:
@timabbott, I have tested this and it works correctly:
```
In [8]: logger = logging.getLogger('django.template')

In [9]: logger.debug("Debug")
2017-02-14 01:28:46,655 DEBUG    Debug

In [10]: logger.warn("Warning")
2017-02-14 01:28:52,614 WARNING  Warning

In [11]: logger.error("Error")
2017-02-14 01:28:59,082 ERROR    Error
```

Even if we set `propagate=True`, it works because 'django' logger doesn't handle any messages by default. If we attach a handler which logs to console to 'django' logger then we will get duplicate records in the console according to https://docs.python.org/2/library/logging.html#logging.Logger.propagate.